src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsWindowMapping.java

```diff
@@ -40,25 +41,34 @@ public static HearingWindow buildHearingWindow(@Valid SscsCaseData caseData) {
 
         return HearingWindow.builder()
             .firstDateTimeMustBe(getFirstDateTimeMustBe())
-            .dateRangeStart(getHearingWindowStart(caseData))
+            .dateRangeStart(getDateRangeStart(caseData, referenceDataServiceHolder))
             .dateRangeEnd(null)
             .build();
     }
 
+    public static LocalDate getDateRangeStart(
+        @Valid SscsCaseData caseData,
+        ReferenceDataServiceHolder referenceDataServiceHolder
+    ) {
+        return isHearingAdjourned(caseData)
+            && referenceDataServiceHolder.isAdjournmentFlagEnabled() // TODO remove when flag enabled
+            ? AdjournmentCalculateDateHelper.getHearingWindowStart(caseData)
+            : getHearingWindowStart(caseData);
+    }
+
     public static LocalDate getHearingWindowStart(@Valid SscsCaseData caseData) {
```

Comment:
I think this change makes the code harder to read

Reply:
reverted